dask/dataframe/operation.py

```diff
@@ -16,17 +16,22 @@
 from dask.optimization import SubgraphCallable
 from dask.utils import apply
 
+if TYPE_CHECKING:
+    # TODO: import from typing (requires Python >=3.10)
+    from typing_extensions import TypeAlias
 
-@dataclass(frozen=True)
-class FrameOperation(CollectionOperation):
+    PartitionKey: TypeAlias = "tuple[str, int]"
```

Comment:
Tests are failing because `PartitionKey` is not defined when `TYPE_CHECKING` is False. What is the correct way to set `PartitionKey` in this fall-back case?

Reply:
I think I don't need an explicit `TypeAlias` here, probably simpler than jumping through these hoops. Pushing a fix now